x-pack/test/rule_registry/security_and_spaces/tests/basic/get_browser_fields_by_feature_id.ts

```diff
@@ -56,7 +56,20 @@ export default ({ getService }: FtrProviderContext) => {
           'logs',
           'uptime',
         ]);
-        expect(Object.keys(browserFields)).to.eql(['base', 'event', 'kibana', 'message']);
+        expect(Object.keys(browserFields)).to.eql([
```

Comment:
Additional fields available on Kibana start because indices are installed in kibana start.